.github/workflows/issue-branch.yml

```
@@ -2,84 +1,113 @@
+          branch: ${{ github.head_ref }}
+          file_pattern: ./packages/design-system/**
+          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
 
-      - name: Create new branch
+  add-reviewer:
+    runs-on: ubuntu-latest
+    needs: build-design-system
+    steps:
+      - name: Extract Issue Number from Branch
+        id: extract_issue
         run: |
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)